src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplateField/PhrasalTemplateField.tsx

```diff
@@ -15,93 +17,114 @@ export const PhrasalTemplateField = ({
-                fill="currentColor"
-                id="drag"
-              />
-            </IconButton>
-          </>
-        ) : (
-          <IconButton color="default" onFocus={handleFocusMenu}>
-            <Svg
-              aria-label={t('phrasalTemplateItem.btnShowFieldControls')}
-              fill="currentColor"
-              id="dots"
-              height={18}
-              width={18}
+    <>
+      <Draggable index={index} draggableId={itemId}>
+        {(draggableProvided, snapshot, rubric) => (
+          <StyledFlexTopCenterDraggable
+            sx={{ gap: 0.8 }}
+            ref={draggableProvided.innerRef}
+            {...draggableProvided.draggableProps}
+            isDragging={snapshot.isDragging && rubric.draggableId === itemId}
+          >
+            <RenderedField
+              name={name}
+              responseOptions={responseOptions}
+              type={type}
+              index={index}
+              itemId={itemId}
+              sx={{ '.MuiFormHelperText-root': { mb: 0.4 } }}
+              {...otherProps}
             />
-          </IconButton>
+            <StyledFlexTopCenter
+              onBlur={hideExpandedMenu}
+              onMouseEnter={showExpandedMenu}
+              onMouseLeave={hideExpandedMenu}
+              ref={menuRef}
+              sx={{ gap: 0.8 }}
+            >
+              {isExpandedMenuOpen ? (
+                <>
+                  <IconButton
+                    color="default"
+                    disabled={!canRemove}
+                    onClick={handleOpenRemoveItemMenu}
+                  >
+                    <Svg
+                      aria-label={t('phrasalTemplateItem.btnRemoveField')}
+                      fill="currentColor"
+                      id="trash"
+                    />
+                  </IconButton>
+                  <IconButton color="default" {...draggableProvided.dragHandleProps}>
+                    <Svg
+                      aria-label={t('phrasalTemplateItem.btnOrderField')}
+                      fill="currentColor"
+                      id="drag"
+                    />
+                  </IconButton>
+                </>
+              ) : (
+                <IconButton
+                  color="default"
+                  onMouseEnter={showExpandedMenu}
+                  onMouseLeave={hideExpandedMenu}
+                  {...draggableProvided.dragHandleProps}
+                >
+                  <Svg
+                    aria-label={t('phrasalTemplateItem.btnShowFieldControls')}
+                    fill="currentColor"
+                    id="dots"
+                    height={18}
+                    width={18}
+                  />
+                </IconButton>
+              )}
+            </StyledFlexTopCenter>
+          </StyledFlexTopCenterDraggable>
         )}
-      </StyledFlexTopCenter>
-    </StyledFlexTopCenter>
+      </Draggable>
+      <Menu
+        anchorEl={removeItemMenuAnchorEl}
```

Comment:
Just a suggestion to match Figma little better:
```suggestion
        anchorEl={removeItemMenuAnchorEl}
        anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
        transformOrigin={{ vertical: 'top', horizontal: 'right' }}
        width="23.2rem"
```